main.go

```diff
@@ -225,34 +200,40 @@ func main() {
 }
 
 func prepareFilter(filters []string) (tracee.Filter, error) {
-
 	filterHelp := "\n--filter allows you to specify values to match on for fields of traced events.\n"
-	filterHelp += "The following options are currently supported:\n"
-	filterHelp += "uid: only trace processes or containers with specified uid(s).\n"
+	filterHelp += "\n"
+	filterHelp += "uint filters use the following operators: '=', '!=', '<', '>'\n"
+	filterHelp += "available uint filters: uid, pid, mntns, pidns\n"
+	filterHelp += "Examples:\n"
```

Comment:
I think it's better to put all the examples at the end. if someone it reading this from start to end they see a block of examples and don't keep reading to see there a more filter types